### ISSUE TITLE
some fontawesome icons are not shown

### ISSUE THREAD

**Issue by joejoe2016** [devlom/grav-theme-hola#34]
Hi,
first of all thanks for your awesome theme. I'm enjoying using it.

Unfortunately I have a problem with some icons that I would like to use - they are not displayed.
In my resume.en.md I use these lines
icon_class: 'fa fa-coffee'
icon_class: 'fa fa-pepper-hot'

The coffee icon is shown, the pepper-hot icon not although they both seem to be free icons on fontawesome [see eg. here](https://fontawesome.com/icons/pepper-hot?style=solid) What am I doing wrong?

What is a bit confusing to me is that in grav the icon is used with fa fa-coffee but on fontawesome with fas fa-coffee,

Btw: I'm using hola version 1.1.2 and grav version v1.6.16.

Thanks
joejoe

**Comment by sebasvisser** [comment]
If you download a newer version and place them in the fonts folder this should be fixed.

Ideally you could switch the theme over to support the online hosted font awesome fonts, instead of the local files. (Although that might make the website a bit slower in some cases..)

tldr; download a newer version of font awesome
